src/browser/browser.js

```diff
@@ -21,7 +21,7 @@ define((require, exports, module) => {
   const {WebViewBox, WebView} = require('./web-view');
   const {Dashboard} = require('./dashboard');
   const {readDashboardNavigationTheme} = require('./dashboard/actions');
-  const {activate: activateStrip, readInputURL,
+  const {activate: activateStrip, readInputURL, sendEventToChrome,
```

Comment:
Artifact of my initial attempt on #32b17d7bd92ad1825f19fa0431815380ffb7f813

Good catch